sound/soc/bcm/allo-piano-dac-plus.c

```diff
@@ -708,7 +708,7 @@ static int snd_allo_piano_dac_init(struct snd_soc_pcm_runtime *rtd)
 	if (!glb_ptr)
 		return -ENOMEM;
 
-	memset(glb_ptr, 0x00, sizeof(glb_ptr));
+	memset(glb_ptr, 0x00, sizeof(*glb_ptr));
```

Comment:
This is correct, but it would be better to replace the `kmalloc` above with `kzalloc` and delete this line.